server/controllers/login/login.js

```diff
@@ -18,13 +18,14 @@ exports.post = (request, response) => {
     })
     .then((isHashed) => {
       if (!isHashed) {
-        response.status(401).send({
-          error: {
-            code: 401,
-            msg: 'Invalid username or password',
-          },
-          data: [],
-        });
+        next({ code: 500 });
```

Comment:
@Angham116 I think it would be best if this error code was still a `401`, since it's handling the case where the posted password does not match the database password.

Reply:
@ashatat Sorry! I was looking at an out-of-date diff.   Nevermind!

Reply:
It's ok @tomduggan85 I didn't see it myself :)